### PR TITLE
Update GitHub URLs and code archiving

### PR DESCRIPTION
**Pull request by dhimmel**

### CODE REVIEW

File: content/response-to-reviewers.md [hunk 1]
```diff
@@ -41,6 +41,11 @@ This a totally minor point, but it might help to attract new people.
 > You need to have a long-term archive (e.g. Zenodo) of manubot (just in case GitHub closes tomorrow).
 Or you can use a Software Heritage identifier to make sure manubot can still be found in a few years.
 
+This feedback was discussed in [GH135](https://github.com/greenelab/meta-review/issues/135) and addressed in [GH159](https://github.com/greenelab/meta-review/pull/159).
```
Comment (agitter) [inline content/response-to-reviewers.md:44]: Do we want to link to issues and pull requests for all responses?  We could, but I haven't done it for the other minor issues I closed.

File: content/response-to-reviewers.md [hunk 2]
```diff
@@ -41,6 +41,11 @@ This a totally minor point, but it might help to attract new people.
...
+We created GitLab mirrors of important Manubot repositories, to provide access to Manubot's latest source code during any GitHub downtime.
+We also added Software Heritage identifiers, to preserve access to Manubot's source code via snapshots.
+We found this preferable to Zenodo archiving because we were not ready to commit to perpetually maintaining Zenodo records that potentially require manual edits and therefore increase the effort required to make releases.
```
Comment (agitter) [inline content/response-to-reviewers.md]: It's not necessary, but perhaps add one more line about how the Manubot release process is currently mostly automated.  It's a very nice approach.

File: content/02.main-text.md [hunk 3]
```diff
@@ -304,3 +304,11 @@ Manubot is an ideal authoring protocol for real-time open science, especially fo
 While Manubot does require technical expertise, the benefits are manyfold.
 Specifically, Manubot demonstrates a system for publishing that is transparent, reproducible, immediate, permissionless, versioned, automated, collaborative, open, linked, provenanced, decentralized, hackable, interactive, annotated, and free of charge.
 These attributes empower integrating Manubot with an ecosystem of other community-driven tools to make science as open and collaborative as possible.
+
+## Code & data availability
```
Comment (agitter) [inline content/02.main-text.md]: I prefer `and` over `&` in a formal manuscript.

File: content/02.main-text.md [hunk 4]
```diff
@@ -304,3 +304,11 @@ Manubot is an ideal authoring protocol for real-time open science, especially fo
...
+
+The source code and data for this manuscript are available at <https://github.com/greenelab/meta-review> and archived via Software Heritage identifier [`swh:1:dir:5e644c3a487081b272b2c9b52bcd55caa89c4f85`](https://archive.softwareheritage.org/swh:1:dir:5e644c3a487081b272b2c9b52bcd55caa89c4f85;origin=https://github.com/greenelab/meta-review/).
+Source code for Manubot resides in the following repositories:
+
++ <https://github.com/manubot/manubot> ([GitLab mirror](https://gitlab.com/manubot/manubot), archived at [`swh:1:dir:1f820c7aa29a5971e7e8e129cc62682131cda19f`](https://archive.softwareheritage.org/swh:1:dir:1f820c7aa29a5971e7e8e129cc62682131cda19f;origin=https://github.com/greenelab/manubot/))
++ <https://github.com/manubot/rootstock> ([GitLab mirror](https://gitlab.com/manubot/rootstock), archived at [`swh:1:dir:dfdf5d0fde7fa32d6559c817f37878de93e32916`](https://archive.softwareheritage.org/swh:1:dir:dfdf5d0fde7fa32d6559c817f37878de93e32916;origin=https://github.com/greenelab/manubot-rootstock/))).
```
Comment (agitter) [inline content/02.main-text.md]: Extra `)` here

File: content/02.main-text.md [hunk 5]
```diff
@@ -304,3 +304,11 @@ Manubot is an ideal authoring protocol for real-time open science, especially fo
...
 Specifically, Manubot demonstrates a system for publishing that is transparent, reproducible, immediate, permissionless, versioned, automated, collaborative, open, linked, provenanced, decentralized, hackable, interactive, annotated, and free of charge.
 These attributes empower integrating Manubot with an ecosystem of other community-driven tools to make science as open and collaborative as possible.
+
+## Code & data availability
+
+The source code and data for this manuscript are available at <https://github.com/greenelab/meta-review> and archived via Software Heritage identifier [`swh:1:dir:5e644c3a487081b272b2c9b52bcd55caa89c4f85`](https://archive.softwareheritage.org/swh:1:dir:5e644c3a487081b272b2c9b52bcd55caa89c4f85;origin=https://github.com/greenelab/meta-review/).
+Source code for Manubot resides in the following repositories:
+
++ <https://github.com/manubot/manubot> ([GitLab mirror](https://gitlab.com/manubot/manubot), archived at [`swh:1:dir:1f820c7aa29a5971e7e8e129cc62682131cda19f`](https://archive.softwareheritage.org/swh:1:dir:1f820c7aa29a5971e7e8e129cc62682131cda19f;origin=https://github.com/greenelab/manubot/))
```
Comment (agitter) [inline content/02.main-text.md]: Should we also add the PyPI link here?

**Review by agitter — CHANGES_REQUESTED** [review]
The new availability section looks great.

The draft pull request feature is interesting.  I believe that I did not receive a separate notification once it was designated as ready for review.

File: content/response-to-reviewers.md (hunk 1 above)
Comment (dhimmel) [reply content/response-to-reviewers.md:44]: Yes, I was imagining we could use a sentence like this on every issue. I think it is helpful for viewers to see exactly where a comment was addressed. In addition, the issue and PRs often have additional comments that provide more information on why we made the changes we did.

**Review by agitter — APPROVED** [review]

**Pull request merged**